tla/consensus/ccfraft.tla

```diff
@@ -1219,13 +1223,18 @@ MonoLogInv ==
 LogConfigurationConsistentInv ==
     \A i \in Servers :
         \/ state[i] = None
+        \* Follower, but no known configurations yet
+        \/ /\ state[i] = Follower
+           /\ Cardinality(DOMAIN configurations[i]) = 0
         \/
             \* Configurations should have associated reconfiguration txs in the log
             /\ \A idx \in DOMAIN (configurations[i]) : 
                 /\ log[i][idx].contentType = TypeReconfiguration            
                 /\ log[i][idx].configuration = configurations[i][idx]
-            \* Current configuration should be committed
-            /\ commitIndex[i] >= CurrentConfigurationIndex(i)
+            \* Current configuration should be committed if there is one
```

Comment:
This just does not hold, and not just at startup, because the highest config in the log is not even committable (and so definitely not committed) until it's followed by a signature. This is what broke in the MC once the change at 1227/8 was made to fix situations where there aren't any configs on the new joiner yet.